server/routerlicious/packages/lambdas/src/moira/lambda.ts

```diff
@@ -129,11 +131,14 @@ export class MoiraLambda implements IPartitionLambda {
     ): Promise<void> {
         const commitGuid = opData.guid;
 
-        this.context.log?.info(
-            `MH Commit: branch: ${branchGuid},
-             commit ${commitGuid},
-             changeSet:  ${JSON.stringify(opData.changeSet, undefined, 2)}`,
-        );
+        const logMessage = `MH Commit: branch: ${branchGuid},
+        commit ${commitGuid},
+        changeSet:  ${JSON.stringify(opData.changeSet, undefined, 2)}`;
+        this.context.log?.info(logMessage);
+
+        if (this.serviceConfiguration.enableLumberMetrics) {
```

Comment:
The idea behind using this flag was to enable and disable the metrics..should we rename it or create a new one if we need it?

Reply:
Done, renamed the flag to be more generic